src/cfnlint/data/CloudSpecs/ap-northeast-1.json

```diff
@@ -52819,6 +52822,25 @@
       "NumberMax": 35,
       "NumberMin": 0
     },
+    "AWS::RDS::DBInstance.Engine": {
+      "AllowedValues": [
+        "aurora",
+        "aurora-mysql",
+        "aurora-postgresql",
+        "mariadb",
+        "mysql",
+        "MySQL",
```

Comment:
I feel like this one is a mistake?  

Also can't find it in Engine here https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_CreateDBInstance.html

Reply:
This template deploys successfully in `us-east-1`:
```yaml
Resources:
  DBInstance:
    Type: AWS::RDS::DBInstance
    DeletionPolicy: Delete
    Properties:
      DBInstanceClass: db.t3.micro
      AllocatedStorage: 10
      Engine: MySQL
      MasterUsername: MasterUsername
      MasterUserPassword: MasterUserPassword
```

I discovered this from this test failing:

https://github.com/aws-cloudformation/cfn-python-lint/blob/d9c7c29ff7eef6bb6830f731bfee27f463b438bb/test/fixtures/templates/quickstart/nist_application.yaml#L1015

and saw [Troposphere had `MySQL` as well](https://github.com/cloudtools/troposphere/blob/d05fed14659013a597f0686d9584215af13da597/troposphere/rds.py#L16)